package.json

```diff
@@ -31,14 +34,18 @@
   },
   "dependencies": {
     "cssmin": "^0.4.3",
+    "minimist": "^1.2.0",
     "node-sass": "^3.1.2"
   },
   "devDependencies": {
+    "cross-spawn": "^2.0.0",
+    "eslint": "^1.9.0",
     "mocha": "2.2.5",
     "proxyquire": "1.6.0",
     "sinon": "1.15.4"
   },
   "scripts": {
-    "test": "NODE_ENV=test mocha"
+    "test": "NODE_ENV=test mocha",
+    "pretest": "eslint *.js bin/scss-to-json src test"
```

Comment:
I'm not sure I follow here. Similar to you `cli-test.js` I run the test in a shell. The test is not run in pre test but as part of the test suite. The pretests is just eslint. (I will merge master into this branch to avoid conflicts.)